c2corg_ui/templates/helpers/view.html

```diff
@@ -111,7 +111,8 @@
       title = ''
       if 'title_prefix' in locale and locale['title_prefix']:
           title = locale['title_prefix'] + ' : '
-      title += locale['title']
+      if 'title' in locale and locale['title']:
+          title += locale['title']
```

Comment:
note - for that wrong version Title field is strange (----) and description/remarks/gear fields are empty http://localhost:6548/routes/111829/fr/291699
``` sql
SELECT * 
FROM 
  guidebook.documents_versions, guidebook.documents, guidebook.documents_archives, 
  guidebook.documents_geometries_archives, guidebook.documents_locales_archives, 
  guidebook.history_metadata, guidebook.routes_locales_archives
WHERE 
  documents_versions.document_id = documents.document_id AND
  documents_versions.document_archive_id = documents_archives.id AND
  documents_versions.document_geometry_archive_id = documents_geometries_archives.id AND
  documents_versions.history_metadata_id = history_metadata.id AND
  documents_locales_archives.id = documents_versions.document_locales_archive_id AND
  routes_locales_archives.id = documents_locales_archives.id AND
  guidebook.documents_versions.document_id = 111829;
```